Makefile

```diff
@@ -294,80 +305,87 @@ fedora_podman:  ## Build Fedora podman development container
 prior-fedora_podman:  ## Build Prior-Fedora podman development container
 	$(call build_podman_container,$@,$(PRIOR_FEDORA_RELEASE))
 
-$(_TEMPDIR)/%_podman.tar: podman/Containerfile podman/setup.sh $(wildcard base_images/*.sh) $(_TEMPDIR) $(wildcard cache_images/*.sh)
-	podman build -t $*_podman:$(call err_if_empty,_IMG_SFX) \
-		--security-opt seccomp=unconfined \
+# N/B: This assumes that qemu_user_static is present and installed into the kernel or (as root)
+# podman run --rm --privileged mirror.gcr.io/multiarch/qemu-user-static:latest --reset -p yes
+# must been executed (e.g. on CentOS).
+$(_TEMPDIR)/%_podman.iid: podman/Containerfile podman/setup.sh $(wildcard base_images/*.sh) $(_TEMPDIR) $(wildcard cache_images/*.sh)
+	  podman build --manifest=$*_podman:$(call err_if_empty,_IMG_SFX) \
+		--net=host --ipc=host --pid=host \
+    --cgroupns=host --security-opt label=disable \
+    --security-opt seccomp=unconfined \
+	  --platform=linux/amd64,linux/arm64 \
```

Comment:
note to me: Fix the gawdawful indenting.  Also, probably don't need `--net=host --ipc=host --pid=host --cgroupns=host`.